ios/google_navigation_flutter/Sources/google_navigation_flutter/ImageRegistry.swift

```diff
@@ -13,6 +13,7 @@
 // limitations under the License.
 
 import Foundation
```

Comment:
Nit: We already discussed this. UIKit internally importing Foundation.

Otherwise LGTM.